analyse_performance.py

```
@@ -0,0 +1,490 @@
+                      "tflow": "tflow.{}",
+                      "tdat": "tdat.{}"}
+
+    # Set the operating conditions from the config
+    oper = copy.deepcopy(config.multi_oper[0])
+
+
+    def __init__(self,
+                 beta_tip: float,
+                 rps_range: list[float] | np.typing.NDArray[np.floating],
+                 verbose: bool = True,
+                 **kwargs) -> None:
```

Ruff (0.14.7)

Unused method argument: `kwargs`

(ARG002)